docs/getting_started.md

```diff
@@ -19,18 +19,17 @@
 - Run `conda remove scipy`
```

Comment:
Thanks for remembering to do both--we need to make that standard practice for everyone. We need some sort of team best practices doc.